### ISSUE TITLE
(Follower) How to prepare my own training data or where is add_data or add_comment comes from?

### ISSUE THREAD

**Issue by angrysword** [ROBOTIS-GIT/turtlebot3_applications#24]
Hi team,
    I used TB3-burger follower. I like to improve the follower classification. However, I dont know how to generate my own training data. thanks

import pickle
import rospy
import matplotlib.pyplot as plt
import numpy as np
from sensor_msgs.msg import LaserScan
from sklearn.ensemble import RandomForestClassifier

if __name__ == '__main__':

    laser_data = pickle.load( open( "add_data", "rb"))
    comments = pickle.load( open( "add_comment", "rb" ))
    labels = {'30_0':0, '30_l':1, '30_r':2, '45_0':3, '45_l':4, '45_r':5, '15_0':6, 'empty':7}

**Comment by kijongGil** [comment]
Hi @angrysword :)
This application uses scikit-learn package.  I attached scikit-learn tutorial.
http://scikit-learn.org/stable/tutorial/basic/tutorial.html#

**Comment by angrysword** [comment]
Hi KijongGil

     Thanks for replying,  the actual question is about where did you get "add_data" file and "add_comment" file? or may I generate my own add_data file for better recognize the legs.

**Comment by robotpilot** [comment]
Hi @angrysword
How are you doing?  I hope everything is going great. If you need it, you have to create your data. Please give me feedback if you've succeeded to do something. 
@kijongGil, can you give him a tip for it if you can?

**Comment by kijongGil** [comment]
I'm sorry to lately reply.
You can get "add_comment" file from laser_subscriber.py and ploting.py.
If you add labels you wanted, you can add more detail LDS data.

Thanks,
Gilbert.

**Comment by angrysword** [comment]
I still have no idea like where did you get the 'add_data' file or how do you generate this 'add_data' file. I am thinking if I can prepare more training data like that, I may come up with a better learning model

**Comment by angrysword** [comment]
thanks very much. I will try it and give you feedback soon.

**Comment by angrysword** [comment]
I got it. that is what I am looking for, I never think laser subscriber will do that jobs, never take a look at it .  Definitely MY FAULT. Thank you very much. I appreciate your time.
